src/resources/SearchAnalysis/SearchAnalysisInterfaces.ts

```diff
@@ -0,0 +1,158 @@
+import {PostSearchBodyQueryParams} from '../Search/SearchInterfaces';
+declare type RequestParameters = PostSearchBodyQueryParams;
+interface Result {
+    title: string;
+    rankingInfo: unknown;
```

Comment:
really? There's not a better type?

Reply:
Not yet. We didn't define how we will type the ranking information but we will eventually define a strict type for it.